### PR TITLE
Product listing

### PR DESCRIPTION
**Pull request by parulgupta2001**

### CODE REVIEW

File: styles/component-style/product-listing.css [hunk 1]
```diff
@@ -0,0 +1,97 @@
+*{
+    --color-one:orange;
+    --color-two:#173F5F;
+}
```
Comment (srishtigautam21) [inline styles/component-style/product-listing.css:4]: you should use var for colors

**Review by srishtigautam21 — APPROVED** [review]
good to merge with suggested changes

File: styles/component-style/product-listing.css (hunk 1 above)
Comment (parulgupta2001) [reply styles/component-style/product-listing.css:4]: var is apply when we use these colors

**Review by Megha-Joshi — APPROVED** [review]
Good to merge

**Pull request merged**